### DISCUSSION TITLE
Naive question about file extension

### DISCUSSION THREAD

**Discussion by stefanos82** [edubart/nelua-lang#78] [Q&A] [answered]
I have been thinking about it since day 1; how come you didn't use `.nel` for file extension?

**Comment by edubart** [comment, marked as answer]
Most file extensions with 3 letters are used already by other applications, so this was avoided to not have conflicts with other softwares. And to be clear in case someone that has never seen the `.nelua` extension could google it with ease and find out what the file is.

Nelua name was also chosen in a special way, I've verified that nothing named Nelua could be found on google and on social networks, this way I can keep track of where Nelua is being used worldwide and its growth. By having the file extension to be also Nelua makes this easier to keep track where Nelua sources is being used.

I personally dislike ungoogleable programming language names like C, Go, Ruby (and sadly Lua), the amount of unrelated results you find when searching about it makes a developer's life harder, as you have to filter many unrelated results in your search and in the end waste more time.

**Reply by stefanos82** [reply]
Yeah, it makes sense from that particular perspective; `nelua`'s uniqueness makes it easier to track usage.
